### ISSUE TITLE
Clarity of examples using MRS

### ISSUE THREAD

**Issue by washcycle** [Microsoft/RTVS-docs#47]
One thing with MRS that confuses me is the separation of the client and the server. It appears all these examples run on an MRS node. I'd like to see clarification of that in the examples and perhaps a commented out section showing how to run it if it were to run it from a Windows and/or Linux client.

**Comment by andrie** [comment]
Right now there is weak separation between R Server and R Client. (For an overview of R Client see https://msdn.microsoft.com/en-us/microsoft-r/r-client-get-started).

All the examples we show will run both using R Server and R Client.

I think your question is how to use R Client to connect to a remote compute context (e.g. Hadoop or SQL Server). You can find information on using compute contexts at https://msdn.microsoft.com/en-us/microsoft-r/scaler-distributed-computing.

**Comment by washcycle** [comment]
Right, I think my suggestion is to note that in the readme's or examples somewhere how to run these examples on a remote compute context. Like so:

```
# For Windows
mySshUsername <- "username"
#name of PuTTY saved session
mySshHostname <- "putty_saved_sessionname"
myShareDir <- paste("/var/RevoShare", mySshUsername, sep = "/")
myHdfsShareDir <- paste("/user/RevoShare", mySshUsername, sep = "/")

myHadoopCluster <- RxSpark(
    hdfsShareDir = myHdfsShareDir,
    shareDir = myShareDir,
    sshUsername = mySshUsername,
    sshHostname = mySshHostname,
    sshClientDir = "c:\\program files (x86)\\putty")
```
